test/extensions/filters/http/transformation/inja_transformer_replace_test.cc

```diff
@@ -173,6 +173,29 @@ TEST(Extraction, AttemptReplaceFromPartialMatchNonNilSubgroup) {
   EXPECT_EQ("", res);
 }
 
+TEST(Extraction, AttemptReplaceFromNoMatchNonNilSubgroup) {
+  Http::TestRequestHeaderMapImpl headers{{":method", "GET"}, {":path", "/foo"}};
+
+  envoy::api::v2::filter::http::Extraction extractor;
```

Comment:
nit: can replace this type definition with `ExtractionApi` as defined [here](https://github.com/solo-io/envoy-gloo/blob/55ddd1bfc4d9976bc953cab2dd0fcc874edd9405/source/extensions/filters/http/transformation/inja_transformer.h#L29)

Reply:
Resolved, w/ other test cleanup here: 202d18dfc7ca07ede7f41ed2f5abefcb9a2d0126